tests/roles/keystone_adoption/defaults/main.yaml

```diff
@@ -6,6 +6,10 @@ keystone_patch: |
       apiOverride:
         route: {}
       template:
+        KeystoneAPI:
```

Comment:
I'm not sure, but it looks to me like the  customServiceConfig should be directly below "template:", so this line is not needed. This is how it works in glance, ex.: https://github.com/openstack-k8s-operators/glance-operator/blob/f66b05484df8df1df9cba1fc7560fbd2f4ce04ae/config/samples/backends/nfs/nfs.yaml#L10

Reply:
correct, this needs to be changed and stay under template.